modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -143,6 +145,31 @@ spec:
       protocol: TCP
     - port: 443
 ----
+
+[id="configuringNodeSelector-example_{context}"]
+== Example nodeSelector for {kind}
+
+As a cluster administrator, you can allow or deny egress traffic to nodes in your cluster by specifying a label using `nodeSelector`. Labels can be applied to one or more nodes. The following is an example with the `type=user-node, region=east`:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: default
+spec:
+  egress:
+  - type: Allow
+    to:
+    nodeSelector:
+      region: east
+      type: user-node
+----
+
+[TIP]
+====
+Rather than adding manual rules per node IP address, use `nodeSelector` to create a label to allow pods behind an egress firewall to access host network pods.
```

Comment:
_Rather than_ sounds odd here to me.  _Instead of_ ?
s/to allow/that allows  (there are a lot of _to_'s in there!)